Study Management update for the help guide

nurikarakaya opened the pull request:
## What?
Update the Study Management section in the help guide.

## Why?
To provide clearer, more accurate, and up-to-date instructions for users managing studies.

## How?
Revised the content  and related files to improve explanations, add missing details, and update screenshots.

## Testing?
Reviewed the rendered site locally to ensure formatting, links, and images display correctly. Checked for typos and broken links.

## Screenshots (optional)


Anything Else?
 Please let me know if further changes are needed.

nurikarakaya committed:
docs: initial commit

nurikarakaya committed:
refactor: add styling for better readablity

nurikarakaya committed:
docs: add screenshots for risk of bias; work on nav-studies

nurikarakaya committed:
docs: add screenshots and amend wording to match with the current ui

nurikarakaya committed:
docs: start building the structure of the Study Management topic

crude structure created

nurikarakaya committed:
docs: update the page; remove repeated parts

nurikarakaya committed:
docs: improve study management page; in progress

nurikarakaya committed:
docs: remove view studies page, already in the study management

nurikarakaya committed:
docs: Improve Study Management page with clarifications and screenshots

nurikarakaya committed:
docs: Consolidate study management docs and fix navigation

Remove duplicate content from nav-studies.md, fix anchor links in
Study Management page, and improve navigation structure. Creates
single source of truth for study management documentation.

nurikarakaya committed:
docs: add What's New entry for enhanced Study Management interface

gcurrie768 commented on whats-new/whats-new-study-management-ui.md:
```diff
@@ -0,0 +1,38 @@
+
+We've unified the management of studies in your SyRF project in the **Study Management** interface. 
+
+The updated interface now features three clearly organised main sections:
```
Remove "clearly organised"
```suggestion
The updated interface now features three main sections:
```

gcurrie768 commented on whats-new/whats-new-study-management-ui.md:
```diff
@@ -0,0 +1,38 @@
+
+The updated interface now features three clearly organised main sections:
+
+* **Systematic Searches Table:** Your primary workspace where you can upload a new systematic search, bulk update existing studies, start risk of bias calculation, and the actions for viewing, deleting and copying the IDs of studies.
```
```suggestion
* **Systematic Searches Table:** Your primary workspace where you can upload a new systematic search, bulk update existing studies including updating screening decisions and adding PDF relative links, initiate automated risk of bias assessment, and view and delete systematic searches from your project.
```

gcurrie768 commented on whats-new/whats-new-study-management-ui.md:
```diff
@@ -0,0 +1,38 @@
+
+* **Systematic Searches Table:** Your primary workspace where you can upload a new systematic search, bulk update existing studies, start risk of bias calculation, and the actions for viewing, deleting and copying the IDs of studies.
+
+* **Risk of Bias Assessment Monitoring:** Dedicated section for tracking Risk of Bias assessment processes with detailed progress reporting and comprehensive error reporting.
```
```suggestion
* **Risk of Bias Assessment:** Dedicated section for tracking Risk of Bias assessment processes with detailed progress reporting and comprehensive error reporting.
```

gcurrie768 commented on whats-new/whats-new-study-management-ui.md:
```diff
@@ -0,0 +1,38 @@
+
+* **Risk of Bias Assessment Monitoring:** Dedicated section for tracking Risk of Bias assessment processes with detailed progress reporting and comprehensive error reporting.
+
+* **Job Progress Monitoring:** Real-time tracking of all background processes including Systematic Search Uploads and Bulk Study Updates, with enhanced error details.
```
```suggestion
* **Job Progress Monitoring:** Real-time tracking of all background processes including Systematic Search Uploads and Bulk Study Updates.
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+# Study Management
+
+This guide covers how to manage studies in your SyRF project. On the Study Management page, project administrators can upload, organise, monitor, and update study references throughout the systematic review process.
```
```suggestion
This guide covers how to manage studies in your SyRF project. From the Study Management page, project administrators can upload, organise, monitor, and update study references throughout the systematic review process.
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+After selecting a project, click the **Studies** button in the left navigation bar:
+
+![Study Management Interface](figs/Fig_study-management-access-1.png){: style="border: 2px solid #ccc; padding:10px;"}
```
Not sure whther these screeneshots are necessary- could we just describe where to click in the text without the screenshots?

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+Then click **Study Management** to access the administrative interface:
+
+![Study Management Interface](figs/Fig_study-management-access-2.png){: style="border: 2px solid #ccc; padding:10px;"}
```
Similarly, not sure this screenshot is necessary.

abannachbrown replied:
If only keeping one screenshot - i would argue the second screenshot it better than the 1st

nurikarakaya replied:
first screenshot deleted. one screenshot looks okay to me, should we remove this as well?

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+> **Calculate Risk of Bias**
+>
+> See section [1.3 Run Risk of Bias Assessment](#run-risk-of-bias-assessment) below for detailed instructions on initiating risk of bias calculations.
```
Should we make it clear that this autoamted RoB assessment? 
```suggestion
> See section [1.3 Run Risk of Bias Assessment](#run-risk-of-bias-assessment) below for detailed instructions on initiating automated risk of bias calculations.
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+The Study Management interface provides you with the tools in the workflow described above. 
+
+{: .note }
+> **Important:** Whilst preparing references and managing PDFs are essential parts of study management in SyRF, these steps are performed outside of the Study Management interface itself. Reference preparation happens before upload using external tools, and PDF management involves coordinating with the SyRF helpdesk after upload. We include these steps in this guide because they are crucial prere ...
```
```suggestion
> **Important:** Whilst preparing references and managing PDFs are essential parts of study management for a SyRF project, these steps are performed outside of the Study Management interface itself. Reference preparation happens before upload to SyRF using external tools, and PDF management involves coordinating with the SyRF helpdesk after upload. We include these steps in this guide because they are crucial prerequisites and follow-up tasks that directly support the study management workflow.
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+### 1. Systematic Searches Table
+Your primary workspace showing all uploaded searches with:
+- **Action buttons**: Upload new searches and bulk update studies
```
Need to make it clear what the bulk update feature allows them to do e.g. update screening decisions, update customIDs and update PDF realtive paths.

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+- **Progress indicators**: Real-time status updates with clickable links to detailed progress
+- **Overflow menu**: View studies, delete systematic searches, run Risk of Bias assessment for a single search, copy the ID of a search
+
+Two main actions in the Systematic Searches table, [Upload Systematic Search](upload-search.html) and [Bulk Study Update](manage-studies.html), have their dedicated pages, so the guides for these are not included here.
```
```suggestion
Access guidance for [Upload Systematic Search](upload-search.html) and [Bulk Study Update](manage-studies.html) at these dedicated user guide sections.
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+{: #delete-search}
+> **⚠️ Warning:** This action cannot be undone. This will remove the systematic search from this project, and all screenings, annotations and outcome data for studies in this search will be lost.
+
+Click the red delete button (trash icon) in the Actions column: 
```
Should we call it an Actions column or not? Not sure what the answer is but maybe check with others in the team!

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+![Study Management Interface](figs/Fig_search-delete-start.png){: style="border: 2px solid #ccc; padding:10px;"}
+
+This will open a dialogue box for confirmation. Type the research name into the text field and click Delete to complete the process:
```
```suggestion
This will open a dialogue box for confirmation. Type the systematic search name into the text field and click Delete to complete the process:
```

gcurrie768 commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+This will open a dialogue box for confirmation. Type the research name into the text field and click Delete to complete the process:
+
+![Study Management Interface](figs/Fig_search-delete-confirmation.png){: style="border: 2px solid #ccc; padding:10px;"}
```
I don't think "research name" is the correct term. Shoudl it be "systematic search name"?

gcurrie768 reviewed (commented):
These sections look great Nuri! I have made a few minor suggestions :)

nurikarakaya committed:
docs: Resolve some of the Gillian's comments. Some left for discussion.

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+# Study Management
+
+This guide covers how to manage studies in your SyRF project. From the Study Management page, project administrators can upload, organise, monitor, and update study references throughout the systematic review process.
+
```
Please provide an overview / table of contents for this page (similar to how you have a numbered list at the top of "Preparing References" and "Upload Systematic Search".

abannachbrown replied:
The table of contents for the sub-pages, that it automatically generated is also at the bottom of the page - can we move it up and give some 'scaffolding text' to differentiate it from the ToC for the "Study Management" page itself.

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+
+## Overview of a Typical Workflow
+
+These are the steps in a typical workflow when working with studies in your SyRF project:
```
Typical workflow implies and order to which users should conduct certain steps. However, this section lists all the features/actions that can be completed with this feature/in this page of SyRF. Perhaps re-word this to remove the association with "workflow"?

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+> See sections [2. Risk of Bias Assessment Monitoring](#risk-of-bias-assessment-monitoring) and [3. Job Progress Monitoring](#job-progress-monitoring) below for information on tracking upload progress and background processes. 
+
+## What is Study Management?
+
```
This is somehow the information that I expected to see under "things you can do on this page" above (what you have called workflow). Under "What is XX feature?", I would expected to see things i can do as a user in this section. Maybe we can combine them someone.

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+The Study Management interface provides you with the tools in the workflow described above. 
+
+{: .note }
+> **Important:** Whilst preparing references and managing PDFs are essential parts of study management for a SyRF project, these steps are performed outside of the Study Management interface itself. Reference preparation happens before upload to SyRF using external tools, and PDF management involves coordinating with the SyRF helpdesk after upload. We include these steps in this guide because the ...
```
All this info in "important" section should be moved under "preparing references", it is too much information here. 
F

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+![Study Management Interface](figs/Fig_search-delete-confirmation.png){: style="border: 2px solid #ccc; padding:10px;"}
+
+
+**1.3 Run Risk of Bias Assessment**
```
I think the Risk of Bias assessment should perhaps have its own "sub-page" in this section - e.g. after "Managing PDFs" in the left nav. There is a lot of information here. - If we move it to a sub-page we can also move the Risk of Bias Assessment Monitoring to this sub-page too.

abannachbrown commented on upload-search.md:
```diff
@@ -71,10 +73,25 @@ Upon successful upload, the new Systematic Search will appear in the list on the
 
 * **[Add PDFs](manage-pdfs.html)** - Link full-text PDFs to your uploaded studies
-* **[View Studies](view-studies.html)** - See your uploaded studies in the project
+* **[View Studies](study-management.html#11-view-studies-in-a-systematic-search)** - See your uploaded studies in the project
```
"View Studies" just takes you to the top of the page - if possible it should take the user to the exact section.

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+Click the ellipsis symbol to access the overflow menu and select the button for copying the search ID:
+![Study Management Interface](figs/Fig_copy-search-id.png){: style="border: 2px solid #ccc; padding:10px;"}
+
+**1.5 Status Indicators**
```
to be honest I don't really understand the numbering on this page... I would have thought that status indicators should be under the Job Progress monitoring?

abannachbrown commented on study-management.md:
```diff
@@ -0,0 +1,152 @@
+- **Action buttons**: Upload new searches and bulk update studies
+- **Bulk operations**: Select multiple searches for Risk of Bias calculations
+- **Progress indicators**: Real-time status updates with clickable links to detailed progress
+- **Overflow menu**: View studies, delete systematic searches, run Risk of Bias assessment for a single search, copy the ID of a search
```
I would have thought that the information presented below in the numbered sections would follow this order - but it does not. I don't think it needs to follow the order of actions, operations, progress indicators, and overflow menu. Perhaps this section can be updated after the info below has been refined.

abannachbrown commented on manage-studies.md:
```diff
@@ -24,11 +24,16 @@ This powerful feature allows Project Administrators to modify data for *existing
+
+![Screenshot showing location of Bulk Study Update option](figs/Fig_Bulk-Study-Update.png)
 
 ### Preparing Your Update File (CSV)
```
Can these 3 sections be numbered? 
e.g. Step 1: Prepare Your Update File, Step 2: Upload & Mapping Process

abannachbrown commented on manage-studies.md:
```diff
@@ -99,4 +104,4 @@ Project Administrators can remove an entire Systematic Search upload and all ass
 * **[Upload Search](upload-search.html)** - How to upload your systematic search
 * **[Managing PDFs](manage-pdfs.html)** - How to add and manage PDF files
-* **[View Studies](view-studies.html)** - How to view your uploaded studies
+* **[View Studies](study-management.html#11-view-studies-in-a-systematic-search)** - How to view your uploaded studies
```
I cannot add a comment to Line 80 - The information on deleting a search is duplicated with section 1.2 on "Study Management" page - we should only have the full information in 1 place, and then reference the other sections. Can the colour of the text "Extreme Warning" be changed to red perhaps? or add an icon?

nurikarakaya committed:
docs(study-management): restructure Risk of Bias content and improve page organization

- Create new calculate-risk-of-bias.md sub-page for Risk of Bias functionality
- Move Risk of Bias assessment content from main page to dedicated sub-page
- Restructure study-management.md with clearer section organization
- Update navigation links and fix anchor IDs for proper internal linking
- Add important notes to prepare-references and manage-pdfs pages
- Improve page layout and content flow based on reviewer feedback

Addresses reviewer comments on content organization and navigation

nurikarakaya committed:
docs(study-management): standardize page formatting and add table of contents

- Add automatic TOC generation to all Study Management pages
- Remove redundant headers that duplicate page titles
- Standardize formatting across study-management sub-pages
- Improve page navigation and consistency

Applied to: study-management, prepare-references, upload-search,
manage-pdfs, manage-studies, and calculate-risk-of-bias pages

nurikarakaya committed:
docs: add section numbering for bulk update instructions and update anchor link for view studies
